src/images.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { sendRequest } from './util';
+
+export default function images(): Object {
+  const baseUrl = 'images-api.nasa.gov';
+  const endpoints = {
+    search: '/search',
+    asset: '/asset',
+    metadata: '/metadata',
+    captions: '/captions',
+  };
+
+  function validateMediaType(mediaType: string): string {
```

Comment:
Yeah but it doesn't throw any errors if you add more, like `image,audio,whatever` I think it's functioning as an "OR" query so it will retrieve all types. If you want to validate, it might be better to check for [a-zA-Z] and commas only? That way they can pass in any values, as long as they are properly formatted